projects/plugins/jetpack/modules/notes.php

```diff
@@ -82,8 +82,9 @@ public function action_init() {
 
 		// Do not show notifications in the Site Editor, which is always in fullscreen mode.
 		global $pagenow;
-		// phpcs:ignore WordPress.Security.NonceVerification
-		if ( 'admin.php' === $pagenow && isset( $_GET['page'] ) && 'gutenberg-edit-site' === $_GET['page'] ) {
+
+		$is_site_editor_page = 'site-editor.php' === $pagenow;
```

Comment:
Ditto for https://github.com/Automattic/jetpack/pull/25281/files#r933668048

Reply:
I guess the old logic was missing a check for `themes.php`? In fact, `admin.php?page=gutenberg-edit-site` seems to always redirect to `themes.php?page=gutenberg-edit-site`.